web/modules/custom/mymodule/js/my_modue.js

```diff
@@ -0,0 +1,5 @@
+(function ($) {
+  $(document).ready(function () {
+    alert("Hello world!");  //here we can add our JS code
+  });
+})(jQuery);
```

Comment:
@anitha-innoraft Why do we have a JS file? Let's remove it please.

Reply:
I have started to write code to do validate form using jquery. Then i stopped soing it as you have instructed us to not use jquery. I have removed it now